proof/refine/RISCV64/Invariants_H.thy

```diff
@@ -2676,19 +3312,6 @@ lemma ex_cte_cap_to'_pres:
 
 section "Relationship of Executable Spec to Kernel Configuration"
 
-text \<open>
-  Some values are set per kernel configuration (e.g. number of domains), but other related
-  values (e.g. maximum domain) are derived from storage constraints (e.g. bytes used).
-  To relate the two, we must look at the values of kernel configuration constants.
-  To allow the proofs to work for all permitted values of these constants, their definitions
-  should only be unfolded in this section, and the derived properties kept to a minimum.\<close>
-
-lemma le_maxDomain_eq_less_numDomains:
-  shows "x \<le> unat maxDomain \<longleftrightarrow> x < Kernel_Config.numDomains"
-        "y \<le> maxDomain \<longleftrightarrow> unat y < Kernel_Config.numDomains"
-  by (auto simp: Kernel_Config.numDomains_def maxDomain_def word_le_nat_alt)
-
-
```

Comment:
This one should stay

Reply:
This block appears in `master` at the very top of the file in ARM and ARM_HYP, but in this position in RISCV64. I moved it to the top in this commit. Where would be best?

Reply:
Ah, I missed that it was already at the top. Then it's fine to leave as is, I don't have a particular preference for position.